### DISCUSSION TITLE
Nominating @anthonydahanne to Java Buildpacks Maintainers Team

### DISCUSSION THREAD

**Discussion by pivotal-david-osullivan** [paketo-buildpacks/feedback#187] [Java Team]
@anthonydahanne has made great contributions to the Java related buildpacks as well as CI tooling. He has also participated in Java related RFC discussions and proposed a [new one](https://github.com/paketo-buildpacks/rfcs/pull/290) for creation of a utility buildpack. I would like to formally nominate him to be added to the maintainer team. 

👍 / 👎

**Comment by pivotal-david-osullivan** [comment]
With enough votes, this passes - congrats @anthonydahanne!

**Reply by dmikusa** [reply]
Congrats!

**Reply by anthonydahanne** [reply]
Thank you !!! 🚀
